src/System.Drawing.Common/src/mono/MonoTODOAttribute.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Mono does not necessary need them. They are nice to have but as you can see they are not really updated.